README.md

```diff
@@ -57,7 +57,8 @@ OFirehose require a configuration file located in `/etc/ofirehose.json`, `~/.ofi
    "/etc/ssl/private/myserver.key". Defaults `null`.
 * **cert** If you're using SSL, the path to the server cert, like
    "/etc/ssl/private/myserver.crt". Defaults `null`.
-* **address** The address to listen on. Defaults server hostname, Use this if you've got some kind of load-balancer or NAS or whatever and your local IP doesn't map to the IP of the hostname.
+* **server** The address on which "publish/subscribe"(`PubSubHubbub`) will happen. Defaults machine `hostname`.
+* **address** The address to listen on. Defaults machine `hostname`, Use this if you've got some kind of load-balancer or NAS or whatever and your local IP doesn't map to the IP of the hostname.
```

Comment:
Nit: "Defaults to machine hostname" (same as above)

Also does this really default to the machine hostname? Or does it default to `server`?